### PR TITLE
Update Windows Voice Assistant Client doc & screen captures 

### PR DESCRIPTION
**Pull request by dargilco**
## Purpose

To reflect current version 1.12

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: clients/csharp-wpf/README.md [hunk 1]
```diff
@@ -47,17 +47,18 @@ Let's review the hardware, software, and subscriptions that you'll need to use t
    ```
     -  Launch Visual Studio 2017 or newer by opening the solution `VoiceAssistantClient.sln`. Build the solution (the default build flavor is Debug x64)
     - Run the executable. For example, for Debug x64 build, this will be the executable: `VoiceAssistantClient\bin\x64\Debug\VoiceAssistantClient.exe`.
-1. When you first run the application, the **Setting** page will open. The first two fields are required (all others are optional):
-    - Enter _Subscription key_. This is your Azure [Speech Services Key](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/get-started)
-    - Enter _Subscription key region_. This is the Azure region of your key in the format specified by the "Speech SDK Parameter" column [in this table](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/regions#speech-to-text-text-to-speech-and-translation) (for example "westus")
-    - Leave the field _Custom commands app Id_ empty (unless you plan to use [Custom Commands](#use-custom-commands))
+1. When you first run the application, the **Setting** page will open. The first three fields are required (all others are optional):
```
Comment (splumber) [inline clients/csharp-wpf/README.md:50]: **Setting** -> *Settings**

File: clients/csharp-wpf/README.md [hunk 2]
```diff
@@ -47,17 +47,18 @@ Let's review the hardware, software, and subscriptions that you'll need to use t
...
     - The default input language is "en-us" (US English). Update the _Language_ field as needed to select a different [language code from the "Speech-to-text" list](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support).
-    - Press _Ok_ when you're done.
-    - Your entries will be saved and populated automatically when you launch the app again.
+    - Press _Save and Apply Profile_ when you're done.
+    - Your entries will be saved under this profile name and will be available when you launch the app again.
 ![Setting page](docs/media/SettingsPage.png)  
 1. Press **Reconnect**. The application will try to connect to your bot via Direct Line Speech channel.
 ![Main Page](docs/media/MainPage.png)
-The message **New conversation started -- type or press the microphone button** will appear below the text bar if the connection succeeded.
+The message **Press the mic button or type to start talking to your bot** will appear below the text bar if the connection succeeded.
```
Comment (splumber) [inline clients/csharp-wpf/README.md:61]: The message is "New conversation started - type or press the microphone button" upon successful connection

File: clients/csharp-wpf/README.md [hunk 3]
```diff
@@ -47,17 +47,18 @@ Let's review the hardware, software, and subscriptions that you'll need to use t
...
+    - Enter _Subscription key region_. This is the Azure region of your key in the format specified by the "Speech SDK Parameter" column [in this table](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/regions#speech-to-text-text-to-speech-and-translation) (for example "westus").
+    - Leave the field _Custom commands app Id_ empty (unless you plan to use [Custom Commands](#use-custom-commands)).
     - The default input language is "en-us" (US English). Update the _Language_ field as needed to select a different [language code from the "Speech-to-text" list](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support).
-    - Press _Ok_ when you're done.
-    - Your entries will be saved and populated automatically when you launch the app again.
+    - Press _Save and Apply Profile_ when you're done.
+    - Your entries will be saved under this profile name and will be available when you launch the app again.
 ![Setting page](docs/media/SettingsPage.png)  
 1. Press **Reconnect**. The application will try to connect to your bot via Direct Line Speech channel.
 ![Main Page](docs/media/MainPage.png)
```
Comment (splumber) [inline clients/csharp-wpf/README.md:60]: This image is incorrect - it is a screen capture before Reconnect has been clicked. 

After pressing reconnect and a successful connection, The Connection Profile: **profilename** is displayed and message text box says "New conversation started - type or press the microphone button" --- also depending on the Bot, Activity Log may have an activity to display

Comment (splumber) [reply clients/csharp-wpf/README.md:60]: Also, the Snip & Sketch tool has a ruler option so your lines can look symmetrical ☺️

File: clients/csharp-wpf/README.md (hunk 2 above)
Comment (dargilco) [reply clients/csharp-wpf/README.md:61]: Oh, you’re right, what I wrote was before you press the Reconnect button. Silly me.

File: clients/csharp-wpf/README.md (hunk 1 above)
Comment (dargilco) [reply clients/csharp-wpf/README.md:50]: Thanks!

File: clients/csharp-wpf/README.md (hunk 3 above)
Comment (dargilco) [reply clients/csharp-wpf/README.md:60]: I replaced the image was one that is captured AFTER the reconnection, and mentioned that the profile name appears at the top. 
I'm still using the only snipping tool! switching to the new scratch & snip I see the ruler. So cool! updated the images.

**Pull request merged**